input/postgres/buffer_cache.go

```diff
@@ -2,24 +2,37 @@ package postgres
 
 import (
 	"context"
+	"fmt"
+	"time"
 
 	"github.com/pganalyze/collector/state"
 	"github.com/pganalyze/collector/util"
 )
 
-// See also https://www.postgresql.org/docs/current/pgbuffercache.html
-const bufferCacheSQL string = `
-SELECT reldatabase, relfilenode, count(*) * current_setting('block_size')::int
-FROM pg_buffercache
-GROUP BY 1, 2`
+const bufferCacheExtensionSQL string = `
+SELECT COALESCE((
+	SELECT nspname
```

Comment:
We should probably be calling `quote_literal(nspname)` here, to make sure we support extensions that are installed in schemas that require quoting (e.g., "this schema has spaces"), but I noticed that our handling of the pg_stat_statements schema doesn't do this either, and I don't think we've heard complaints, so we can leave this and fix both in a follow-up patch instead.